R/resource_create.R

```diff
@@ -38,22 +39,24 @@
 #'                        description = "my resource",
 #'                        name = "bears",
 #'                        upload = file,
+#'                        extras = list(species = "grizzly"),
 #'                        rcurl = "http://google.com"
 #' ))
 #'
 #' package_create("foobbbbbarrrr") %>%
 #'    resource_create(description = "my resource",
 #'                    name = "bearsareus",
 #'                    upload = file,
+#'                    extras = list(my_extra = "some value")
```

Comment:
you're missing a comma at the end of this line

Reply:
thx